.gitignore

```diff
@@ -2,4 +2,7 @@
 .idea
 build
 cover.*
-node_modules
\ No newline at end of file
+node_modules
+
+validate
+internal/validate
```

Comment:
I think we don't need this line.

See this [snippet](https://github.com/kevinmichaelchen/api-go-template/blob/bcd3384493b31a8606751350cc9acf41d153e0bf/Makefile#L6-L13).

We remove `validate` before we generate. We only bring it back (with `buf export` once the generation is complete).

Reply:
now i got it. yep, will use the example from the snippet

Reply:
here is the resulting buf-gen
```
rm -rf api/{google,grafeas,validate} || true
	buf mod update api
	buf export buf.build/envoyproxy/protoc-gen-validate -o api

	# Proto files and types from google and grafeas
	# buf export buf.build/googleapis/googleapis -o api

	# Not tracking breaking changes just yet, uncomment after first major release.
	# buf breaking --against '.git#branch=main'
	buf lint --exclude-path api/validate/validate.proto
	buf format -w
	buf generate
```